src/wfs/src/test/java/org/geoserver/wfs/response/ShapeZipTest.java

```diff
@@ -972,7 +972,7 @@ public void testMultiPolygonZMShp() throws Exception {
      * @return shp file byte array
      */
     private byte[] getShpOnlyBytes(byte[] zipBytes) throws IOException {
-        byte[] resultBytes = new byte[] {};
+        byte[] resultBytes = {};
```

Comment:
not sure about this, I may be wrong.
but i think `byte[] resultBytes;` would be sufficient here as default initializer would create 0-length array of byte because it's a primitive (can't check atm)

Reply:
I think I'm wrong about that, reading https://docs.oracle.com/javase/tutorial/java/nutsandbolts/arrays.html